pkg/dynamicinstrumentation/ditypes/config.go

```diff
@@ -17,10 +17,11 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 
-	"github.com/DataDog/datadog-agent/pkg/dynamicinstrumentation/ratelimiter"
 	"github.com/cilium/ebpf"
 	"github.com/cilium/ebpf/link"
 	"github.com/google/uuid"
+
+	"github.com/DataDog/datadog-agent/pkg/dynamicinstrumentation/ratelimiter"
```

Comment:
This should be grouped with `"github.com/DataDog/datadog-agent/pkg/util/log"` right?